data-final-fixes.lua

```diff
@@ -1,3 +1,18 @@
+local lib=require("lib.lib")
+local ps = require("lib.planet-str")
+require("prototypes.override-final.starmap")
```

Comment:
This line introduces a bug. `require("prototypes.override-final.starmap")` now appears further down the page, as is necessary.